src/signals/settings/users/Overview/__tests__/Overview.test.js

```diff
@@ -335,8 +335,8 @@ describe('signals/settings/users/containers/Overview', () => {
 
     expect(dispatch).not.toHaveBeenCalled();
 
-    const filterByUserName = await findByTestId('filterUsersByUsername');
-    const filterByUserNameInput = filterByUserName.querySelector('input');
+    let filterByUserName = await findByTestId('filterUsersByUsername');
```

Comment:
Prefor to use separate tests instead of `let`s, `unmount`s and `rerender`s.

Reply:
We generally don't use `let` in the code. The only exception is in some test scenarios where the initialization is done in the `beforeEach` function.